packages/signature-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.4]
+### Changed
+- Bump dependency on `@metamask/keyring-controller` to ^9.0.0
```

Comment:
@Gudahtt Should this be a breaking change as well?

Reply:
Yes I think so. This is actually miscategorized right now as a dev dependency, it should be a dependency. It's used in the main module, not in tests.

Maybe someone thought that it was dev because it was a type-only dependency, but we consider types as part of the API for a package. And it definitely matters if they change shape.

I'll put up a separate PR to re-categorize this

Reply:
https://github.com/MetaMask/core/pull/2053